### PR TITLE
Updated divisions to math.div

### PR DESCRIPTION
**Pull request by wiredsis**
## Summary

- Updated deprecated divisions for future compatibility (Dart Sass 2.0)

### CODE REVIEW

File: src/client/pages/reversi/game.board.vue [hunk 1]
```diff
@@ -388,11 +388,11 @@ export default defineComponent({
 				font-size: 0.8em;
 
 				&:first-child {
-					margin-left: -($gap / 2);
+					margin-left: -math.div($gap, 2);
 				}
 
 				&:last-child {
-					margin-right: -($gap / 2);
+					margin-right: -math.div($gap / 2);
```
Comment (syuilo) [inline src/client/pages/reversi/game.board.vue]:
```suggestion
					margin-right: -math.div($gap, 2);
```
?

**Comment by wiredsis** [comment]
Don't merge yet. Math doesn't seem to be found!

**Comment by wiredsis** [comment]
I just realized updating all of this is not really painful. I'll add the remaining files that need to be fixed to this pull request and that should be all.

**Comment by wiredsis** [comment]
Ok. Should be ready now.

**Comment by wiredsis** [comment]
Ok. Now it's definitely fixed.

**Comment by wiredsis** [comment]
Checks have all passed! Ready to merge.

**Comment by syuilo** [comment]
Thanks!

**Pull request merged**